app/questionnaire/questionnaire_store_updater.py

```diff
@@ -339,15 +379,26 @@ def update_answers(
             self._current_question
         )
 
+        answers_updated = None
```

Comment:
Minor, default should reflect the actual type.
```suggestion
        answers_updated = False
```

Reply:
Updated 👍